_posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md

```diff
@@ -13,18 +13,32 @@ author: Valentino Urbano
 
 # The Beginning
 
-I'm turning my "Migrating From WordPress To Jekyll" blog post into a small ebook. I will be revisiting each entry and edit it for the medium.
+*I'm turning my "Migrating From WordPress To Jekyll" blog post into a small ebook. I will be revisiting each entry and editing it for the medium.*
 
-Firstly I had to decide how to publish it. I decided to release it for free online, while at the same time release a pdf and epub version on both Amazon and Gumroad for a small fee (still not decided). I don't expect to get any sale, but it is going to be an interesting process to go through nonetheless both generating and formatting the book as well as figuring out how to set up a page on both Amazon and Gumroad.
+Firstly I had to decide how to publish it. I decided to release it online for free, while at the same time releasing a pdf and epub version on both Amazon and Gumroad for a small fee (still not decided). I don't expect to get any sales, but it is going to be an interesting process to go through nonetheless both generating and formatting the book as well as figuring out how to set up a page on both Amazon and Gumroad.
 
 *None of this is set in stone so it might change in the future.*
 
-In the next few days I'll be writing on how to setup a page for a book on any Jekyll based websites.
+In the next few days I'll be writing on how to setup a page for a book on any Jekyll based website.
 
-Writing 200 words a day is great for writing, but I found that I did not have enough time to edit so I am walking a few steps down that both both with [code review fro writing][1] which is open to everyone and with this initiative. I solved the problem on now being able to get enough ideas, but I now have the new problem of feeling like my articles are not edited well due to time concerns. I go more in depth on that topic [in this article][2] so I won't prolong this too much.
+## The Concept Behind It
 
-I have been writing this particular series for a while and it is one of me longest articles. I still have a few entries in it to finish what my baseline idea was for it, but I only did really small and rough edits to it while writing and I didn't have the bigger picture of the whole finished piece before so now it can be edited further and improved. I have a lot going on at the moment so I'm not going to assure any specific timing for it, but as with everything you will be able to follow the progress on GitHub.
+Writing 200 words every day is great for generating new ideas and getting into the habit of writing something on a consistent basis.
+Writing so often means that I did not have enough time to edit since I was writing and publishing an article on the same day.
 
+I will be now moving to write one day in advance as explained in the [code review for writing][1] article. The review process is open for everyone to join and comment so feel free to let me know of any mistakes or improvements that could be made.
```

Comment:
I will be now changing my schedule to write one day (...)?

Reply:
Way better